test/transcoding/KernelArgTypeInOpString2.ll

```diff
@@ -23,21 +23,31 @@
 ; OpString "kernel_arg_type.%kernel_name%.typename0,typename1,..."
 
 ; RUN: llvm-as %s -o %t.bc
-; RUN: llvm-spirv %t.bc -spirv-text -o %t.spv.txt
+; RUN: llvm-spirv -preserve-ocl-kernel-arg-type-metadata-through-string %t.bc -spirv-text -o %t.spv.txt
 ; RUN: FileCheck < %t.spv.txt %s --check-prefix=CHECK-SPIRV
```

Comment:
Again, this should be renamed to something that makes it clearer that this is nonstandard behavior, e.g. CHECK-SPIRV-WORKAROUND.

Reply:
renamed in the second patch